components/datadog/agent/kubernetes.go

```diff
@@ -1,61 +1,29 @@
 package agent
 
 import (
-	"github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 
-	"github.com/DataDog/test-infra-definitions/common/config"
 	"github.com/DataDog/test-infra-definitions/components"
-	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
 )
 
 type KubernetesAgentOutput struct {
 	components.JSONImporter
 
-	InstallNameLinux   string `json:"installNameLinux"`
-	InstallNameWindows string `json:"installNameWindows"`
+	NodeAgent     map[string]string `json:"nodeAgent"`
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Did you check if the export works fine? I am familiar with exporting strings, so I am curious to see if this works fine. Not sure we can test it on this side, maybe with unit, but this can be deferred to a follow up from our team.

Reply:
It outputs like this: 

Outputs Preview:
```
Outputs:
    dd-Cluster-kind                     : output<string>
    dd-Fakeintake-aws-kind              : output<string>
    dd-Host-aws-kind                    : output<string>
    dd-KubernetesAgent-aws-datadog-agent: output<string>
```

and the NodeAgent output:
```
    dd-KubernetesAgent-aws-datadog-agent: {
        clusterAgent : {
            installAppVersion: "7"
            installVersion   : "3.68.2"
            kind             : "Pod"
            labelSelectors   : (json) {
                app: "dda-linux-datadog-cluster-agent"
            }

            name             : "dda-linux-clusterAgent"
            namespace        : "datadog"
        }
        clusterChecks: {
            installAppVersion: "7"
            installVersion   : "3.68.2"
            kind             : "Pod"
            labelSelectors   : (json) {
                app: "dda-linux-datadog-clusterchecks"
            }

            name             : "dda-linux-clusterChecks"
            namespace        : "datadog"
        }
        nodeAgent    : {
            installAppVersion: "7"
            installVersion   : "3.68.2"
            kind             : "Pod"
            labelSelectors   : (json) {
                app: "dda-linux-datadog"
            }

            name             : "dda-linux-nodeAgent"
            namespace        : "datadog"
        }
    }
```